runironic.sh

```diff
@@ -2,6 +2,8 @@
 
 . /bin/configure-ironic.sh
 
+crudini --set /etc/ironic/ironic.conf DEFAULT host localhost
```

Comment:
Just to note, this will functionally restrict the services to only run as a single pair on a single host because [DEFAULT]host is what is used to register the location of where to send the RPC messages. While not an issue now, it will likely become an issue at some point.

Reply:
I think that restriction is acceptable for the time being.